dll/shellext/deskadp/deskadp.c

```diff
@@ -308,28 +308,48 @@ InitDisplayAdapterDialog(PDESKDISPLAYADAPTER This)
         LocalFree((HLOCAL)lpAdapterName);
     }
 
+    WCHAR lpNA[64];
```

Comment:
Move just below `LPTSTR lpAdapterName;` above. (and TCHAR)

Reply:
ok